src/Components/TimelineNotes/TimelineNotes.scss

```diff
@@ -36,7 +36,7 @@
   }
 }
 .notes-container {
-  height: 65vh !important;
+  height: 75vh !important;
```

Comment:
I have also tested your changes and I believe it's not necessary to have the `!important` rule.